src/pages/Entity/WorkOrderInfo.tsx

```diff
@@ -54,9 +59,13 @@ const WorkOrderInfo = ({
                 }
             >
                 <TagInfoWrapper>
-                    <CollapsibleCard cardTitle="Description">
-                        <p>{workOrder?.description}</p>
-                    </CollapsibleCard>
+                    {workOrder?.description && (
```

Comment:
Vi har prøvd å holde oss til  _condition? true : false_ istedenfor _conditon &&_ pga. litt rare truthy verdier (som []) men her er det ikke strengt tatt nødvendig.

Reply:
Det er noen fallgruver med &&, selv om det ser bedre ut, så vi bestemte oss for å gjøre ternery til standard. Kent C Dodds skrev en artikkel om det her: https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx, så får du se om du er enig @Noggling

Reply:
hvis det er grunne så syns jeg den er tynn.

Reply:
@ErlendLygre @VRiveland

Reply:
Grunnen er jo at vi ender opp med å ikke rendre noe som helst dersom verdien vi ville sjekke om var undefined/null, tilfeldigvis er satt til 0 eller "". Det har skjedd meg flere ganger (vi begynte med && i Lærdal, men endte opp med å bytte), så jeg er ikke enig at grunnen er syltynn. Som regel går det bra med &&, men ikke alltid, og da synes jeg det best med ternary overalt. Fordelen med && er at vi slipper å skrive ": null" . Jeg synes fordelen med ternary er større. @Noggling

Reply:
@ErlendLygre Da kan vi være enig om at vi er uenige. Men jeg skal bøye meg for flertallet og det som allerede er satt som standard.